<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FCUwindow</class>
 <widget class="QWidget" name="FCUwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1170</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="FCU_Display_Frame">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>0</y>
     <width>861</width>
     <height>91</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <widget class="QLabel" name="label_SPD">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>13</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>170</green>
          <blue>85</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>170</green>
          <blue>85</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>69</red>
          <green>69</green>
          <blue>69</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SPD</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_HDG">
    <property name="geometry">
     <rect>
      <x>177</x>
      <y>13</y>
      <width>40</width>
      <height>22</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>170</green>
          <blue>85</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>170</green>
          <blue>85</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>69</red>
          <green>69</green>
          <blue>69</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>HDG</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ALT">
    <property name="geometry">
     <rect>
      <x>509</x>
      <y>13</y>
      <width>40</width>
      <height>22</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>170</green>
          <blue>85</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>170</green>
          <blue>85</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>69</red>
          <green>69</green>
          <blue>69</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ALT</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_VS">
    <property name="geometry">
     <rect>
      <x>714</x>
      <y>13</y>
      <width>40</width>
      <height>22</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>170</green>
          <blue>85</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>170</green>
          <blue>85</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>69</red>
          <green>69</green>
          <blue>69</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>V/S</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdn_SPD">
    <property name="geometry">
     <rect>
      <x>48</x>
      <y>33</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>170</green>
          <blue>85</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>170</green>
          <blue>85</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>69</red>
          <green>69</green>
          <blue>69</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="digitCount">
     <number>4</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
    <property name="intValue" stdset="0">
     <number>888</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdn_HDG">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>33</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>170</green>
          <blue>85</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>170</green>
          <blue>85</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>69</red>
          <green>69</green>
          <blue>69</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="digitCount">
     <number>3</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
    <property name="intValue" stdset="0">
     <number>888</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdn_ALT">
    <property name="geometry">
     <rect>
      <x>537</x>
      <y>33</y>
      <width>110</width>
      <height>41</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>170</green>
          <blue>85</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>170</green>
          <blue>85</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>69</red>
          <green>69</green>
          <blue>69</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="digitCount">
     <number>5</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
    <property name="intValue" stdset="0">
     <number>88888</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdn_VS">
    <property name="geometry">
     <rect>
      <x>716</x>
      <y>33</y>
      <width>110</width>
      <height>41</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>170</green>
          <blue>85</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>170</green>
          <blue>85</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>69</red>
          <green>69</green>
          <blue>69</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="digitCount">
     <number>5</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
    <property name="intValue" stdset="0">
     <number>-8888</number>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="FCU_Buttons_Frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>93</y>
     <width>961</width>
     <height>231</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <widget class="QDial" name="dial_HDG">
    <property name="geometry">
     <rect>
      <x>251</x>
      <y>27</y>
      <width>110</width>
      <height>110</height>
     </rect>
    </property>
    <property name="maximum">
     <number>30</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="sliderPosition">
     <number>0</number>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="wrapping">
     <bool>true</bool>
    </property>
    <property name="notchesVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QDial" name="dial_ALT">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>27</y>
      <width>110</width>
      <height>110</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="sliderPosition">
     <number>0</number>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="wrapping">
     <bool>true</bool>
    </property>
    <property name="notchesVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QDial" name="dial_SPD">
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>27</y>
      <width>110</width>
      <height>110</height>
     </rect>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="sliderPosition">
     <number>0</number>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="wrapping">
     <bool>true</bool>
    </property>
    <property name="notchesVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QDial" name="dial_VS">
    <property name="geometry">
     <rect>
      <x>797</x>
      <y>27</y>
      <width>110</width>
      <height>110</height>
     </rect>
    </property>
    <property name="maximum">
     <number>30</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="sliderPosition">
     <number>0</number>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="wrapping">
     <bool>true</bool>
    </property>
    <property name="notchesVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QSlider" name="slider_ALT_SELECT">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>21</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1</number>
    </property>
    <property name="sliderPosition">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="button_athr">
    <property name="geometry">
     <rect>
      <x>453</x>
      <y>151</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>57</width>
      <height>57</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="button_loc">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>160</y>
      <width>68</width>
      <height>59</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>54</width>
      <height>45</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="button_ap1">
    <property name="geometry">
     <rect>
      <x>416</x>
      <y>80</y>
      <width>68</width>
      <height>59</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>54</width>
      <height>45</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="button_ap2">
    <property name="geometry">
     <rect>
      <x>497</x>
      <y>80</y>
      <width>68</width>
      <height>59</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>54</width>
      <height>45</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="button_exped">
    <property name="geometry">
     <rect>
      <x>621</x>
      <y>160</y>
      <width>68</width>
      <height>59</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>54</width>
      <height>45</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="button_appr">
    <property name="geometry">
     <rect>
      <x>817</x>
      <y>160</y>
      <width>68</width>
      <height>59</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>54</width>
      <height>45</height>
     </size>
    </property>
   </widget>
   <widget class="QDial" name="dial_switch_SPD_MACH">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>7</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="maximum">
     <number>2</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
    <property name="sliderPosition">
     <number>1</number>
    </property>
    <property name="wrapping">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDial" name="dial_switch_HDG_TRK">
    <property name="geometry">
     <rect>
      <x>457</x>
      <y>7</y>
      <width>66</width>
      <height>66</height>
     </rect>
    </property>
    <property name="maximum">
     <number>2</number>
    </property>
    <property name="sliderPosition">
     <number>1</number>
    </property>
    <property name="wrapping">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDial" name="dial_switch_MET_ALT">
    <property name="geometry">
     <rect>
      <x>721</x>
      <y>7</y>
      <width>66</width>
      <height>66</height>
     </rect>
    </property>
    <property name="maximum">
     <number>2</number>
    </property>
    <property name="sliderPosition">
     <number>1</number>
    </property>
    <property name="wrapping">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>320</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="fcu_rc.qrc">:/FCUrc/Images/FCU_background.jpg</pixmap>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>FCU_Display_Frame</zorder>
  <zorder>FCU_Buttons_Frame</zorder>
 </widget>
 <resources>
  <include location="fcu_rc.qrc"/>
 </resources>
 <connections/>
</ui>
